tests/phpunit/tests/rest-api/rest-posts-controller.php

```diff
@@ -223,6 +226,21 @@ public function test_registered_get_item_params() {
 		$this->assertSame( array( 'context', 'id', 'password' ), $keys );
 	}
 
+	public function test_registered_get_items_embed() {
```

Comment:
Missing `@ticket` annotation.

```suggestion
	/**
	 * @ticket 43439
	 */
	public function test_registered_get_items_embed() {
```